ksim/actuators.py

```diff
@@ -88,10 +94,40 @@ def get_ctrl(self, action: Array, physics_data: PhysicsData) -> Array:
         """Get the control signal from the (position) action vector."""
         current_pos = physics_data.qpos[7:]  # First 7 are always root pos.
         current_vel = physics_data.qvel[6:]  # First 6 are always root vel.
+        action = action * self.position_scale
```

Comment:
i would strongly prefer that the logic for this lives wherever the task is being implemented - adding this here makes it more difficult to deploy the policy later, whereas if we can put the logic into the task itself it will be clearer and hopefully force us to simplify how this is implemented